lib/node_modules/@stdlib/_tools/changelog/parse-commits/lib/commits.js

```diff
@@ -110,13 +113,20 @@ function extractCommits( dir ) {
 	cmd = join( __dirname, '..', 'scripts', 'commits.sh' );
 	commits = [];
 	args = [ resolve( dir ) ];
+	if ( flags ) {
+		flagNames = objectKeys( flags );
+		for ( i = 0; i < flagNames.length; i++ ) {
+			args.push( '--'+flagNames[i]+'='+flags[ flagNames[ i ] ] );
```

Comment:
```suggestion
			args.push( '--'+flagNames[ i ]+'='+flags[ flagNames[ i ] ] );
```

We should use consistent spacing here.